core/src/test/kotlin/org/stellar/anchor/sep10/Sep10ServiceTest.kt

```diff
@@ -562,6 +562,33 @@ internal class Sep10ServiceTest {
     assertThrows<SepException> { sep10Service.createChallenge(cr) }
   }
 
+  @Test
+  fun testRequireKnownObnibusAccount() {
```

Comment:
The typo I mentioned before is also in all these tests names. I'm very guilty of falling into this same copy/paste error 😅.